src/sscg_generator_cli.erl

```diff
@@ -129,24 +128,39 @@ serialize_arg(ArgName, Args, ArgSpec) ->
                 binary ->
                     io_lib:format("-~s ~s", [LongOpt, binary_to_list(Value)]);
                 {custom, _ParseFun} when ArgName == authors ->
-                    io_lib:format("-~s ~s", [LongOpt, binary_to_list( serialize_authors(Value))]);
+                    io_lib:format("-~s ~s",
```

Comment:
You should make sure you are handling Unicode properly everywhere, to not mess up characters outside of the Latin-1 range like `ő`. This includes using the `~ts` format specifier and using the `utf8` type in binaries.
https://www.erlang.org/doc/apps/stdlib/unicode_usage.html